.github/workflows/alpha-integration-tests.yaml

```diff
@@ -0,0 +1,92 @@
+name: DataRepo Alpha Integration Tests
+env:
+  GOOGLE_APPLICATION_CREDENTIALS: /tmp/jade-dev-account.json
+  GOOGLE_SA_CERT: /tmp/jade-dev-account.pem
```

Comment:
If you changes these to just "jade-dev-account.json" & "jade-dev-account.pem", then should only need to write the vault tokens once.

Reply:
The vault tokens are written twice to the same location because one activation is to get the alpha SA for cleaning up the IAM policies, whereas the other activation is to run integration with the dev SA.

Reply:
Nothing is running in containers so `/tmp` is usable.